src/util/headers.ts

```diff
@@ -35,8 +35,8 @@ export enum Header {
 
   /**
    * ```txt
-   * ETag: W/"<etag_value>";
-   * ETag: "<etag_value>";
+   * ETag: W / '<etag_value>';
+   * ETag: '<etag_value>';
```

Comment:
ha !! this is not right. prettyfied should be disabled for this section

Reply:
For some reason, the prettier is doing this automatically. However if I add some text before it to format like this. IDK why.

Reply:
This is not modified by prettier. what are you using to prettify ?

Reply:
It doesn't even have a semicolon at the end, it must have been inserted some time ago :)